modules/tinymce/src/plugins/insertdatetime/test/ts/browser/InsertDatetimeSanityTest.ts

```diff
@@ -1,43 +1,38 @@
-import { ApproxStructure, Log, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader, TinyUi } from '@ephox/mcagar';
+import { ApproxStructure } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import Editor from 'tinymce/core/api/Editor';
 
-import InsertDatetimePlugin from 'tinymce/plugins/insertdatetime/Plugin';
-import SilverTheme from 'tinymce/themes/silver/Theme';
+import Plugin from 'tinymce/plugins/insertdatetime/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.plugins.insertdatetime.InsertDatetimeSanityTest', (success, failure) => {
+describe('browser.tinymce.plugins.insertdatetime.InsertDatetimeSanityTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
```

Comment:
Can this be moved to a `before` or `beforeEach`?

Reply:
The hook needs to sit inside the `describe` like this to work

Reply:
Also worth noting that `TinyHooks` is an implementation that uses the before, etc... hooks, so it's already using that and this is how you access the data setup in those hooks without having to redeclare the same TinyMCE setup logic everytime (ie reduces duplication).